day2/day2.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import "projects/day2/unique"
+
+func main() {
+	a := []int{1, 1, 1, 1, 2, 2, 2}
+	var b []int
+	b = unique.UniqueArr(a)
```

Comment:
You don't have to create a variable before assigning smth. Try: 
```suggestion
	b := unique.UniqueArr(a)
```